client/a8c-for-agencies/sections/migrations/primary/migrations-commissions/index.tsx

```diff
@@ -34,16 +35,28 @@ export default function MigrationsCommissions() {
 		setShowAddSitesModal( true );
 	}, [ dispatch ] );
 
-	const { data: migrationCommissions, isFetching: isFetchingCommissions } =
-		useFetchMigrationCommissions();
-
-	if ( isFetchingCommissions ) {
-		// TODO: Add a loading state
-		return null;
-	}
+	const { data: migrationCommissions, isFetched } = useFetchMigrationCommissions();
 
 	const showEmptyState = ! migrationCommissions?.length;
 
+	let content = (
```

Comment:
Nitpicking: I recently learned that storing a render element without useMemo causes a repaint for all elements, even when there are no changes to the state. Perhaps consider wrapping the content with useMemo. What do you think?

Reply:
Thanks for the review, @jkguidaven!

Makes sense. I have addressed your feedback: https://github.com/Automattic/wp-calypso/pull/96390/commits/a071cadf7e9d3b1705396ce6fe9122f787963f60